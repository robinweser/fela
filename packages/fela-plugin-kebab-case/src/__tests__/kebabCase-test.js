import kebabCase from '../index'

describe('kebab-case plugin', () => {
  it('should normalize properties', () => {
    const style = {
      color: 'blue',
      'background-color': 'red',
    }

    expect(kebabCase()(style)).toMatchSnapshot()
  })

  it('should normalized nested objects', () => {
    const style = {
      color: 'blue',
      ':hover': {
        'background-color': 'red',
      },
    }

    expect(kebabCase()(style)).toMatchSnapshot()
  })
})
