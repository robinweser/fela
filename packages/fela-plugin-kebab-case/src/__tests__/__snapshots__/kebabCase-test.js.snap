// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`kebab-case plugin should normalize properties 1`] = `
{
  "backgroundColor": "red",
  "color": "blue",
}
`;

exports[`kebab-case plugin should normalized nested objects 1`] = `
{
  ":hover": {
    "backgroundColor": "red",
  },
  "color": "blue",
}
`;
